pkg/clusterapi/config/containerd_config_append.toml

```diff
@@ -1,10 +1,12 @@
 [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."public.ecr.aws"]
-    endpoint = ["https://{{.registryMirrorAddress}}"]
+    endpoint = ["https://{{index .registryMirrorAddressMappings "public.ecr.aws"}}"]
```

Comment:
why is `public.ecr.aws` hardcoded here? shouldn't it be part of the passed mappings?

Reply:
this is just used to retrieve the value for that particular key from the mapping. Also, `public.ecr.aws` and `783794618700.dkr.ecr.*.amazonaws.com` are production registry for eksa artifacts and curated packages respectively. They are extremely unlikely to change.